app/core/application.coffee

```diff
@@ -55,6 +55,11 @@ Application = {
 
     # propagate changes from global 'me' User to 'me' vuex module
     store = require('core/store')
+
+    routerSync = require('vuex-router-sync')
+    vueRouter = require('app/core/vueRouter').default()
+    routerSync.sync(store, vueRouter)
```

Comment:
I assume that there is no reason to unsync because `application` is a global singleton.

Reply:
Yup